### DISCUSSION TITLE
Daily/Weekly/Monthly Summary

### DISCUSSION THREAD

**Discussion by greenpasta** [iSoron/uhabits#836] [Feature Requests]
I'd love the ability to see a combined summary of all habits for a day, week, month, etc. Something like the percentage of my habits that I've completed over that period of time. Completing every habit every day would be ideal, but if I'm not quite there yet it would be cool to see if I'm doing better this week than last week.

**Comment by RedApCi** [comment]
This would be fantastic. Some page with a general overview, or maybe even a widget with a summary. An overall score would be great, maybe even with customisable weighting on the importance of each habit.

**Comment by iSoron** [comment]
This is related to #51.
